ci/README.md

```diff
@@ -0,0 +1,98 @@
+# VIP Go CI Scripts
+
+This directory is not deployed to your VIP site.
+
+## `deploy.sh` and `deploy-exclude.txt`
+
+Scripts designed to facilitate building your Javascript and CSS
+on either [Travis CI](https://travis-ci.com) or [Circle CI](https://circleci.com/).
+
+See our documentation here: (NEEDS LINK, Ed)
+
+### [Circle CI](https://circleci.com/)
+
+Sample Circle CI config, put into `.circleci/config.yml`:
```

Comment:
We should call out the `steps` section (i.e. "configure your build steps"), since it's easy to gloss over. Otherwise, we'll have devs adding the config as-is and wondering why things don't just work :)

Reply:
Addressed in https://github.com/Automattic/vip-go-skeleton/pull/20/commits/559ce1fc966f3e3160ee65f9c94238401fe75295